dev/packages/alpha/mysql-0.0.2/manifest.yml

```diff
@@ -0,0 +1,56 @@
+format_version: 1.0.0
+name: mysql
+title: MySQL
+version: 0.0.2
+license: basic
+description: MySQL Integration
+type: integration
+categories:
+- logs
+- metrics
+release: ga
+requirement:
+  kibana:
+    versions: '>=7.3.1 <8.0.0'
+  elasticsearch:
+    versions: '>7.0.1'
+screenshots:
+- src: /img/kibana-mysql.png
+  title: kibana mysql
+  size: 1227x1026
+  type: image/png
+- src: /img/metricbeat-mysql.png
+  title: metricbeat mysql
+  size: 2562x2540
+  type: image/png
+icons:
+- src: /img/logo_mysql.svg
+  title: logo mysql
+  size: 32x32
+  type: image/svg+xml
+datasources:
+- name: mysql
+  title: MySQL logs and metrics
+  description: Collect logs and metrics from MySQL instances
+  inputs:
+  - type: logs
+    description: Collecting MySQL logs
```

Comment:
I think it would be nice to also have a title for the logs and metrics input

Reply:
Fixed.